programs/programs/policyengine/engines.py

```
@@ -47,39 +47,40 @@ def members(self, unit, sub_unit):
-        res = requests.post(self.domain + self.endpoint, json=payload)
+    if not _pe_client_id or not _pe_client_secret:
+        raise Exception("Policy Engine client id or secret not configured")
 
-        return res.json()["access_token"]
+    payload = {
+        "client_id": _pe_client_id,
+        "client_secret": _pe_client_secret,
+        "grant_type": "client_credentials",
+        "audience": "https://household.api.policyengine.org",
+    }
+    res = requests.post(_pe_token_url, json=payload)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)